python/ray/_private/runtime_env/agent/main.py

```diff
@@ -97,18 +99,16 @@ def open_capture_files(log_dir):
         "--logging-rotate-bytes",
         required=False,
         type=int,
-        default=ray_constants.LOGGING_ROTATE_BYTES,
+        default=sys.maxsize,
```

Comment:
The behavior here is tricky, we have two types of files here: runtime_agent.out and runtime_agent.log.

In terms of current behavior, default rotation size for `out` file is `sys.maxsize`;
while default value for `log` file is `ray_constants.LOGGING_ROTATE_BYTES`.

Reply:
By reading the code, I think this is the option we already set so default should be never used. We can just change `required=True` so we don't need to worry about the default.

Reply:
I mean, when user / dev construct the runtime env agent, they have to fill in the argument.

Reply:
Anyway I updated `required` as you requested